### PR TITLE
feat: Remove check for edgex-kong

### PR DESCRIPTION
**Pull request by bnevis-i**
This is part of the microservice token authentication ADR

### CODE REVIEW

**Comment by bnevis-i** [comment]
@farshidtz Here is another change that is required before I can merge the breaking change for

File: test/utils/net.go
Comment (bnevis-i) [reply test/utils/net.go:75]: It would be 8000 and 8443.  And add proxy-auth microservice as well.  I have a PR for adding the checks back in.  https://github.com/canonical/edgex-snap-testing/pull/164

Postgres (5432) is actually the offending service that is being removed.

Comment (farshidtz) [reply test/utils/net.go:75]: Got it, thanks.
So we can merge this now and then work on #164 once https://github.com/edgexfoundry/edgex-go/pull/4244 is merged.

**Review by farshidtz — APPROVED** [review]

File: test/utils/net.go
Comment (bnevis-i) [reply test/utils/net.go:75]: 👌

**Pull request merged**
